documentation/src/main/wiki/Server.md

```diff
@@ -1,37 +1,93 @@
 # Server
+
+Server is essentially a "heart", a main [Verticle](http://vertx.io/docs/vertx-core/java/#_verticles) of Knot.x.
+It creates HTTP Server, listening for browser requests, and is responsible for coordination of communication between [[Repository|Repository]], [[Splitter|Splitter]] and all deployed [[Knots|Knot]].
+
+## How it works
+[TBD]
+
+## Configuration
+Configuration of the Server is being supplied in Knot.x configuration JSON as for other verticles. The `config` section looks like the one below
 ```json
 {
-  ...
-  "com.cognifide.knotx.server.KnotxServerVerticle": {
-    "config": {
-      "http.port": 8092,
-      "allowed.response.headers": [
-        "User-Agent",
-        "X-Solr-Core-Key",
-        "X-Language-Code",
-        "X-Requested-With"
-      ],
-      "repositories": [
-        {
-          "path" : "/content/local/.*",
-          "address" : "knotx.core.repository.filesystem"
+  "http.port": 8080,
+  "allowed.response.headers": [ "Content-Type", "Content-Length", "Location", "Set-Cookie" ],
+  "repositories" : [
+    {
+      "path": "/content/local/.*\.html",
+      "address": "filesystem.repository.eventbus.address"
+    },
+    {
+      "path": "/content/.*\.html",
+      "address": "http.repository.eventbus.address"
+    }
+  ],
+  "splitter": {
+    "address" : "splitter.eventbus.address"
+  },
+  "routing": {
```

Comment:
Maybe it would be good idea to present this configuration also on a diagram here?